### PR TITLE
Add new api requests https://jiraeu.epam.com/browse/EPMRPP-61999

### PR DESCRIPTION
**Pull request by astraszab**

### CODE REVIEW

**Comment by astraszab** [comment]
search by pattern test: 
image

**Comment by astraszab** [comment]
log indexing test:
image
image

**Comment by astraszab** [comment]
update policy test:
image
image
image

File: commons/es_client.py [hunk 1]
```diff
@@ -188,3 +185,54 @@ def search_logs(self, search_query):
...
+                        "value": search_query["query"],
+                    }
+                }
+            }
+        })
+
+    def index_logs(self, index_query):
+        logs = index_query["logs"]
+        project_id = index_query["project"]
+        index_name = self.format_index_name(project_id)
```
Comment (MariyaIvanina) [inline commons/es_client.py]: when index was not created you need to create on your own and with index template, and let's create a standard policy to rollover after 1 week and move to the warm phase after 2 weeks, but delete will be added only when update policy is called. So you need to add logic here with templates and policies.

**Review by MariyaIvanina — CHANGES_REQUESTED** [review]

**Comment by astraszab** [comment]
initializing ilm test:
image
image
image
image
image

File: commons/es_client.py (hunk 1 above)
Comment (astraszab) [reply commons/es_client.py]: done

**Review by MariyaIvanina — APPROVED** [review]

**Pull request merged**